pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/equation/EquationEvaluator.java

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.ddp.equation;
+
+import ch.obermuhlner.math.big.BigDecimalMath;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import org.broadinstitute.ddp.pex.lang.EquationBaseVisitor;
+import org.broadinstitute.ddp.pex.lang.EquationParser;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.util.Map;
+
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public final class EquationEvaluator extends EquationBaseVisitor<BigDecimal> {
+    private final Map<String, BigDecimal> variables;
+
+    public static EquationEvaluatorBuilder builder() {
+        return new EquationEvaluatorBuilder();
+    }
+
+    @Override
+    public BigDecimal visitPower(final EquationParser.PowerContext ctx) {
+        return BigDecimalMath.pow(visit(ctx.expression(0)), visit(ctx.expression(1)), MathContext.DECIMAL64);
+    }
+
+    @Override
+    public BigDecimal visitMultiplication(final EquationParser.MultiplicationContext ctx) {
+        return visit(ctx.expression(0)).multiply(visit(ctx.expression(1)), MathContext.UNLIMITED);
+    }
+
+    @Override
+    public BigDecimal visitDivision(final EquationParser.DivisionContext ctx) {
+        return visit(ctx.expression(0)).divide(visit(ctx.expression(1)), MathContext.DECIMAL64);
```

Comment:
In the case of addition, subtraction, multiplication you can't produce significantly different values (I mean with a lot of digits after comma). In case of division or power you can. So with the UNLIMITED context, we'll have an exception, so we need to specify the limit for the result.

Reply:
We can use DECIMAL64 everywhere. What do you think?

Reply:
Ahhh- I see. Things like `1/3` will cause `UNLIMITED` precision a great deal of unhappiness. Let's stick with `DECIMAL64` across the board for now

Reply:
Yeah, exactly. Ok, I will change it to DECIMAL64